ci/run.sh

```diff
@@ -32,6 +32,19 @@ else
     $run --features no-f16-f128 --release
 fi
 
+if [ "${TEST_VERBATIM:-}" = "1" ]; then
+    verb_path=$(cmd.exe //C echo \\\\?\\%cd%\\testcrate\\target2)
+    run="cargo build --manifest-path testcrate/Cargo.toml --target $target --target-dir $verb_path"
+    $run
+    $run --release
+    $run --features c
```

Comment:
Only `$run --features c` is really needed here. Let's avoid wasting too much CI time on an edge case.